### PR TITLE
[10.0][NEW] sale_layout_hidden_section module

### PR DESCRIPTION
**Pull request by ilmir-k**
[10.0][NEW] sale_layout_hidden_section module

### CODE REVIEW

**Comment by ilmir-k** [comment]
@yelizariev @Railia it requires your approves

File: sale_layout_hidden_section/README.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+============================
+ Sale Layout Hidden Section
+============================
+
+This module allows to use layouts (sections) within specific Sale Order only.
```
Comment (yelizariev) [inline sale_layout_hidden_section/README.rst]: It's not clear which layouts is about, what is default behaviour

File: sale_layout_hidden_section/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Artyom Losev
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": """Hidden Layouts for Sale Orders""",
+    "summary": """
+        Use layouts within specific Sale Order only""",
```
Comment (yelizariev) [inline sale_layout_hidden_section/__manifest__.py]: Neither name nor summary explains what is for clearly

**Review by yelizariev — CHANGES_REQUESTED** [review]
Too difficult to understand what is for. No picture of results, only how to configure it

**Comment by yelizariev** [comment]
default

**Pull request merged**